### PR TITLE
Add command 'gcp verify wif-config'

### PR DESCRIPTION
**Pull request by JakobGray**
Issue: https://issues.redhat.com/browse/OCM-12286

```bash
➜  ocm gcp verify wif-config -h
Verify a workload identity federation configuration (wif-config) object.

Usage:
  ocm gcp verify wif-config [ID|Name] [flags]

Flags:
  -h, --help   help for wif-config

➜  ocm gcp list wif-configs                                                           
ID                                DISPLAY NAME  PROJECT ID
2fdn79v1eg7btv5qlakanc4sq8917ut6  jdg           sda-ccs-1 

# Verify wif-config - passes
➜  ocm gcp verify wif-config jdg
WIF configuration is valid

# Modified wif-config (role removed from service account jdg-gcp-pd-csi-driver-op-7ut6) - fails
➜  ocm gcp verify wif-config jdg
Error: verification failed with error: missing role 'compute.storageAdmin'
Running 'ocm gcp update wif-config' may fix errors related to cloud resource misconfiguration.
exit status 1

# Self repair
➜  ocm gcp update wif-config jdg

# Verify again - passes
➜  ocm gcp verify wif-config jdg
WIF configuration is valid

```

### CODE REVIEW

File: cmd/ocm/gcp/verify-wif-config.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+	// Verify the WIF configuration is valid
+	response, err := connection.ClustersMgmt().V1().GCP().WifConfigs().WifConfig(wif.ID()).Status().Get().Send()
+	if err != nil {
+		return errors.Wrapf(err, "failed to verify wif-config")
+	}
+	if !response.Body().Configured() {
+		return errors.Wrapf(errors.New(response.Body().Description()), "failed to verify wif-config")
+	}
+
```
Comment (ckandag) [inline cmd/ocm/gcp/verify-wif-config.go:47]: when configured==true(backend returns 200) pls consider printing some txt to indicate verification succeeded

File: cmd/ocm/gcp/verify-wif-config.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		return errors.Wrapf(err, "failed to get wif-config")
+	}
+
+	// Verify the WIF configuration is valid
+	response, err := connection.ClustersMgmt().V1().GCP().WifConfigs().WifConfig(wif.ID()).Status().Get().Send()
+	if err != nil {
+		return errors.Wrapf(err, "failed to verify wif-config")
+	}
+	if !response.Body().Configured() {
+		return errors.Wrapf(errors.New(response.Body().Description()), "failed to verify wif-config")
```
Comment (ckandag) [inline cmd/ocm/gcp/verify-wif-config.go:45]: pls consider adding some help txt to indicate what the user can do when verification fails

Comment (JakobGray) [reply cmd/ocm/gcp/verify-wif-config.go]: I added the following help text
```
➜  ocm-cli git:(OCM-12286) ✗ go run ./cmd/ocm gcp verify wif-config otto
Error: verification failed with error: missing role 'compute.storageAdmin'
Running 'ocm gcp update wif-config' may fix errors related to cloud resource misconfiguration.
exit status 1
```

File: cmd/ocm/gcp/verify-wif-config.go (hunk 1 above)
Comment (JakobGray) [reply cmd/ocm/gcp/verify-wif-config.go:54]: I added the following text
```
➜  ocm-cli git:(OCM-12286) ✗ go run ./cmd/ocm gcp verify wif-config otto
WIF configuration is valid
```

**Review by ckandag — APPROVED** [review]

**Review by renan-campos — APPROVED** [review]

**Pull request merged**